seqr/views/apis/data_manager_api.py

```diff
@@ -39,6 +39,27 @@ def elasticsearch_status(request):
             client, 'nodes', ['name', 'heap.percent'], filter_rows=lambda node: node['name'] in disk_status):
         disk_status[node.pop('name')].update(node)
 
+    indices, seqr_index_projects = _get_es_indices(client)
```

Comment:
so the diff here is showing really weird, but essentially I pulled out a lot of logic into a `_get_es_indices` helper method, and then everything else showing in this block of green is actually unchanged, and all the red below showing those blocks is also unchanged. Other than pullng out the helper method, the only thing that changed in this file is I added the new `delete_index` method

Reply:
Moving the definition of `_get_es_meta` to the location after `_get_es_indices` may help. But, anyway, they don't matter.